.github/workflows/untrustedPR.yml

```
@@ -149,23 +157,28 @@ jobs:
               }
             }
 
-            if (missing_license !== "" || deprecated_license !== "") {
-              var license_output = '';
-              license_output += 'You have modified or added at least one vcpkg.json where you should check the \"license\" field.\n'
-              if (missing_license !== "") {
-                license_output += 'If you feel able to do so, please consider adding a "license" field to the following files:\n'
-                license_output += missing_license
-                license_output += "\n\nValid values for the license field can be found at https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#license\n\n"
-              }
-              if (deprecated_license !== "") {
-                license_output += 'If you feel able to do so, please consider replacing the deprecated license identifiers in the following files:\n'
-                license_output += deprecated_license
-                license_output += "\n\nDeprecated and non deprecated license identifiers can be found at https://spdx.org/licenses/#deprecated\n"
-              }
+            if (missing_license !== "" || deprecated_license.length > 0) {
+              core.summary.addRaw('You have modified or added at least one vcpkg.json where you should check the \"license\" field.');
+              core.summary.addEOL();
+            }
 
+            if (missing_license !== "") {
+              license_output = 'If you feel able to do so, please consider adding a "license" field to the following files:\n'
+              license_output += missing_license
+              license_output += "\n\nValid values for the license field can be found at https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#license\n\n"
               core.warning(license_output);
             }
 
+            for (let line of deprecated_license) {
+              [file, line_number, match] = line.split(':');
+              var license_output = `The license identifier ${match} is deprecated. `;

```

```suggestion
              var license_output = `The license identifier ${match} is deprecated in favor of another license identifier. For example, LGPL2.1 was deprecated in favor of LGPL2.1-Only or LGPL2.1-Or-Later `;
```

Or similar; as written it sounds like we're deprecating SDPX license identifiers

---

```suggestion
              var license_output = `SPDX deprecated license identifier ${match}. Please check https://spdx.org/licenses/ for an updated identifier.`;

```